### ISSUE TITLE
Failed to start on v2.13.1 with docker & postgres

### ISSUE THREAD

**Issue by AkashRajpurohit** [umami-software/umami#2906]
### Describe the Bug

The container is no longer starting up on v2.13.1.
Reverting to v2.12.1 works fine.

### Database

PostgreSQL

### Relevant log output

```shell
umami_postgres  |
umami_postgres  | PostgreSQL Database directory appears to contain a database; Skipping initialization
umami_postgres  |
umami_postgres  | 2024-08-24 07:43:17.104 UTC [1] LOG:  starting PostgreSQL 15.8 on aarch64-unknown-linux-musl, compiled by gcc (Alpine 13.2.1_git20240309) 13.2.1 20240309, 64-bit
umami_postgres  | 2024-08-24 07:43:17.104 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
umami_postgres  | 2024-08-24 07:43:17.104 UTC [1] LOG:  listening on IPv6 address "::", port 5432
umami_postgres  | 2024-08-24 07:43:17.120 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
umami_postgres  | 2024-08-24 07:43:17.136 UTC [28] LOG:  database system was shut down at 2024-08-24 07:42:43 UTC
umami_postgres  | 2024-08-24 07:43:17.203 UTC [1] LOG:  database system is ready to accept connections
umami           | yarn run v1.22.19
umami           | error Could not write file "/app/yarn-error.log": "EACCES: permission denied, open '/app/yarn-error.log'"
umami           | error An unexpected error occurred: "/app/package.json: Unexpected end of JSON input".
umami           | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
umami           | yarn run v1.22.19
umami           | yarn run v1.22.19
umami           | error Could not write file "/app/yarn-error.log": "EACCES: permission denied, open '/app/yarn-error.log'"
umami           | error Could not write file "/app/yarn-error.log": "EACCES: permission denied, open '/app/yarn-error.log'"
umami           | error An unexpected error occurred: "/app/package.json: Unexpected end of JSON input".
umami           | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
umami           | error An unexpected error occurred: "/app/package.json: Unexpected end of JSON input".
umami           | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
umami exited with code 0
umami           | yarn run v1.22.19
umami           | error Could not write file "/app/yarn-error.log": "EACCES: permission denied, open '/app/yarn-error.log'"
umami           | error An unexpected error occurred: "/app/package.json: Unexpected end of JSON input".
umami           | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
umami exited with code 0
umami           | yarn run v1.22.19
umami           | error Could not write file "/app/yarn-error.log": "EACCES: permission denied, open '/app/yarn-error.log'"
umami           | error An unexpected error occurred: "/app/package.json: Unexpected end of JSON input".
umami           | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
umami exited with code 0
```

### Which Umami version are you using? (if relevant)

v2.13.1

### How are you deploying your application? (if relevant)

Docker

**Comment by mikecao** [comment]
Are you building your own Docker image or pulling the pre-built image?

**Comment by AkashRajpurohit** [comment]
Pulling the pre-built image.

**Comment by mikecao** [comment]
The pre-built image does not come with postgres in the image. The image also doesn't run yarn so it must be something with unique to your image.

**Comment by AkashRajpurohit** [comment]
Isn't this a valid image `ghcr.io/umami-software/umami:postgresql-v2.12.1`? I have been using pre-built image from GitHub registry from quite sometime and it works fine.

> The image also doesn't run yarn

The project uses yarn right, I did not get this 🤔

**Comment by franciscao633** [comment]
Just following back on this. I tried your compose file and everything is working. 

![image](https://github.com/user-attachments/assets/ce4fc716-ff6c-4c52-b5cd-d6212880781d)

It's hard to troubleshoot, since your umami container is returning a permissions error instead of an actual error. Are you able to get Umami working with just a connection string to your postgres?

```
version: '3'
services:
  umami:
    image: 'docker.umami.is/umami-software/umami:postgresql-v2.13.1'
    container_name: 'umami'
    ports:
      - '3010:3000'
    environment:
      DATABASE_URL: DATABASE URL HERE
      DATABASE_TYPE: postgresql
      TRACKER_SCRIPT_NAME: script.js
      COLLECT_API_ENDPOINT: /api/send
      DISABLE_TELEMETRY: 1
      APP_SECRET: secret
    restart: always
    healthcheck:
      test: ['CMD-SHELL', 'curl http://localhost:3000/api/heartbeat']
      interval: 5s
      timeout: 5s
      retries: 5
```

**Comment by AkashRajpurohit** [comment]
Thanks for following up @franciscao633, so this is what I tried now, I upgraded to the new latest version of 2.13.2 with the same setup that I have and now it works fine. This is the image being used: `ghcr.io/umami-software/umami:postgresql-v2.13.2`

I still did not get the reason for failure on 2.13.1 😢 
But for now this is fixed for me on 2.13.2, so closing this issue.
